ci/datamgr.py

```diff
@@ -184,6 +189,30 @@ def impala_create_test_database(con, env):
 ALL_SCHEMAS = collections.ChainMap(PARQUET_SCHEMAS, AVRO_SCHEMAS)
 
 
+@click.group()
+@click.option("-v", "--verbose", count=True)
+def cli(verbose):
+    codes = {0: logging.ERROR, 1: logging.INFO, 2: logging.DEBUG}
+    logger.setLevel(codes.get(verbose, logging.DEBUG))
+
+
+@cli.command()
+@click.pass_context
```

Comment:
Is the usage `python ci/datamgr.py create` here? If yes, can we use a more descriptive command?

Reply:
Done.